catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/SeekbarsPickerWrapper.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick.utils;
+
+import android.support.test.espresso.action.ViewActions;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.uiespresso.util.wrappers.ViewInteractionWrapper;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+public final class SeekbarsPickerWrapper extends ViewInteractionWrapper {
+	public static final float MAX_COLOR_SEEKBAR_VALUE = 255.0f;
```

Comment:
im not a huge fan of mixing the phiro and color picker stuff both in this class. 
if someone ever wants to add some tests to the preview color in the color picker fragment, etc this is going to get messy.
please split those into two wrappers, you can have the common functionality (save, discard, undo) in a common baseclass.